benchmarks/NonStiffODE/LotkaVolterra_wpd.jmd

```diff
@@ -9,7 +9,7 @@ author: Chris Rackauckas
 The purpose of this problem is to test the performance on easy problems. Since it's periodic, the error is naturally low, and so most of the difference will come down to startup times and, when measuring the interpolations, the algorithm choices.
 
 ```julia
-using OrdinaryDiffEq, ParameterizedFunctions, ODE, ODEInterfaceDiffEq, LSODA,
```

Comment:
won't you need to remove the solvers that use it too?

Reply:
I believe all the solvers we are using from it are in ODEInterfaceDiffEq (but I'll double check).